polylogue/cli/sync.py

```
@@ -381,12 +401,14 @@ def run_sync_cli(args: SimpleNamespace, env: CommandEnv) -> None:
             )
 
     if provider in {"chatgpt", "claude"}:
-        exports_root = env.config.exports.chatgpt if provider == "chatgpt" else env.config.exports.claude
+        if getattr(args, "base_dir", None):
+            exports_root = Path(getattr(args, "base_dir")).expanduser()
```

Ruff (0.15.9)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)